plugins/pubIds/doi/locale/en_US/locale.xml

```diff
@@ -48,6 +48,9 @@
 	<message key="plugins.pubIds.doi.manager.settings.doiReassign.description">If you change your DOI configuration, DOIs that have already been assigned will not be affected. Once the DOI configuration is saved, use this button to clear all existing DOIs so that the new settings will take effect with existing objects.</message>
 	<message key="plugins.pubIds.doi.manager.settings.doiReassign.confirm">Are you sure you wish to delete all existing DOIs?</message>
 
+	<message key="plugins.pubIds.doi.manager.settings.doiAssignJournalWide">Assign DOIs</message>
+	<message key="plugins.pubIds.doi.manager.settings.doiAssignJournalWide.description">This function enables you to assign the DOIs to all published journal objects that have no DOI assigned yet. It can't be used with the individual suffix configuration. Please save the wished configuration here first, before this function is used. Note that it can take a long time, depending on the number of the pulished objects in the journal.</message>
+	<message key="plugins.pubIds.doi.manager.settings.doiAssignJournalWide.confirm">Are you sure you wish to assign DOIs to all published objects that have no DOI assigned yet?</message>
```

Comment:
Are you sure you wish to assign DOIs to all published objects that have not been assigned DOIs?